action/gen.go

```diff
@@ -63,6 +64,33 @@ func (gen *GenOpts) LocalRun() error {
 		Data: eData,
 	})
 
+	/*
+		attempt to extract providers from valuesToRender
+
+		if providers are available, create the respective metrics files from the
+		templates
+	*/
+	var providers []string
+	if rawProviders, ok := c.Values["providers"]; ok {
```

Comment:
Please define a use a constant instead of hardcoding strings. This simplifies inconsistencies and simplifies changes.

Reply:
I'm not really sure when we should use constants vs hardcoded strings. I try to use constants if there is a string that I need to use multiple times. To clarify, this is not one of those instances.

Should we use constants for all strings? Or whenever we reuse strings? Or whenever we need to access a struct field?